### ISSUE TITLE
Week 12: More machine learning

### ISSUE THREAD

**Issue by ealouie** [gw-sd-2016/NewsTextAnalysis#2]
@cctoombs @twood02 

This week I finished compiling my training set (13f2f91571cf1618090670d9331f08d212f17837) and started testing out different algorithms in Weka. I decided on Naive Bayes because it had slightly better performance than other algorithms I tried. I've also been watching a lot of tutorial videos on Weka and machine learning so I can take full advantage of the program and improve my results. For example, I have been experimenting with different feature selection algorithms in order to get the best model. Additionally, I compiled a test set (e2351a26a0a15ac74c43bce88f55e6361d2590f3) to use to test my model.
